Demo_Project/src/pages/Register.tsx

```diff
@@ -0,0 +1,172 @@
+import Popover from "@material-ui/core/Popover";
+import { RouteChildrenProps } from "react-router";
+import axios from "axios";
+const useStyles = makeStyles(theme => ({
+  root: {
+    flexGrow: 1,
+    textAlign: "center"
+  },
+  appBar: {
+    backgroundColor: "#ffffff",
+    maxWidth: "100^%",
+    boxShadow: "none"
+  },
+  menuButton: {
+    marginRight: theme.spacing(2)
+  },
+  abeld: {
+    color: "0d0d0d"
+  },
+  disabeld: {
+    color: "#b2aaaa"
+  },
+  title: {
+    flexGrow: 1,
+    fontFamily: "Titillium Web",
+    fontSize: 50
+  },
+  subTitle: {
+    color: "#5CB85C"
+  },
+  textField: {
+    width: "400px"
+  },
+  submitButton: {
+    backgroundColor: "#5CB85C",
+    color: "white",
+    width: "100px",
+    height: "50px"
+  },
+  error: {
+    color: "red"
+  },
+  typography: {
+    padding: theme.spacing(2)
+  }
+}));
+function Register() {
+  const classes = useStyles();
+  const [anchorEl, setAnchorEl] = React.useState(null);
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [username, setUsername] = useState("");
+  const open = Boolean(anchorEl);
+  const id = open ? "simple-popover" : undefined;
+  const handelsignup = (event: any) => {
+    if (!email.includes("@")) {
+      setAnchorEl(event.currentTarget);
+      console.log("sdah");
+    }
+    const body = {
+      user: {
+        username: username,
+        email: email,
+        password: password
+      }
+    };
+    console.log("name=" + username);
+    console.log("email=" + email);
+    console.log("password=" + password);
+    axios
+      .post(`https://conduit.productionready.io/api/users`, body)
+      .then(res => {
+        console.log("res=" + res);
+        console.log(res.data);
+      });
+    //callbackFromParent({flage:false,userName:"reem"});
+  };
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+  return (
+    <div className={classes.root}>
+      <div className={classes.title}>Sing up</div>
+      <div>
+        <Link className={classes.subTitle} to="/login">
+          Have an acount?
+        </Link>
+      </div>
+      <div>
+        <TextField
+          className={classes.textField}
+          id="standard-name"
+          label="Username"
+          type="username"
+          name="username"
+          autoComplete="username"
+          margin="normal"
+          variant="outlined"
+          value={username}
+          onChange={event => {
+            setUsername(event.target.value);
+          }}
+        />
+      </div>
+      <div>
+        <TextField
+          className={classes.textField}
+          id="outlined-email-input"
+          label="Email"
+          type="email"
+          name="email"
+          autoComplete="email"
+          margin="normal"
+          variant="outlined"
+          value={email}
+          onChange={event => {
+            setEmail(event.target.value);
+          }}
+        />
+      </div>
+      <Popover
```

Comment:
move this to a separate component and you can call it `tooltip` or anything descriptive